packages/uikit/src/atoms/Button/index.tsx

```diff
@@ -47,6 +48,12 @@ export const BaseButton = styled.button<ButtonProps>`
         pointer-events: none;
         cursor: disabled;
       `}
+
+    ${small &&
+      css`
+        height: 40px;
```

Comment:
Can I be a _massive_ pain in the ass and say that we should define dimensions in the theme?
we've got the default buttonHeight in there so maybe something like this?
```js
dimensions: {
  button: {
    small: {
     height: '40px',
     padding: '0 15px'
    },
   default: {
     height: '44px',
     padding: '0 25px'
   }
  }
}
```

Reply:
👍